docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -39,6 +39,79 @@ The Fairlearn package is most applicable to two kinds of harms:
+
+- **Teacher effectiveness** is an example of an unobservable theoretical construct. Common measurement models include student performance on standardized exams and qualitative feedback for the teacher’s students.
+
+- **Socioeconomic status** is an example of an unobservable theoretical construct. A common measurement model includes annual household income. 
+
+- **Patient benefit** is an example of an unobservable theoretical construct. A common measurement model involves patient care costs.
+
+**Note:** We cite several examples of unobservable theoretical constructs and measurement models for the purpose of explaining the key terms outlined above.  Please reference Measurement and Fairness for more detailed examples.
+ 
+What is Construct Validity? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Though the Measurement and Fairness paper explores both Construct reliability and Construct validity, we focus our 
+exploration below on Construct Validity. We note that both play an important role in understanding fairness in sociotechnical 
+contexts. With that said, Measurement and Fairness offers a fairness-oriented conceptualization of construct validity, that 
+is helpful in thinking about fairness in sociotechnical contexts. We capture the idea in seven key parts that when combined 
+can serve as a framework for analyzing an AI task and attempting to establish construct validity:
+
+1. **Face validity** – On the surface, how plausible does the measurements produced by the measurement model look?
+
+2. **Content validity** – This has three subcomponents:
+
+   a. **Contestedness** – Is there a single understanding of the unobservable theoretical construct? Or is that understanding contested (and thus context        dependent).
+   b. **Substantive validity** – Can we demonstrate that the measurement model contains the observable properties and other unobservable 			   theoretical constructs related to the construct of interest (and only those)?
+   c. **Structural validity** – Does the measurement model appropriately capture the relationships between the construct of interest and the measured           observable properties and other unobservable theoretical constructs?
+
+3. **Convergent validity** – Do the measurements obtained correlate with other measurements (that exist) from 
+   measurement models for which construct validity has been established? 
+ 
+4. **Discriminant validity** – Do the measurements obtained for the construct of interest correlate with 
+   related constructs as appropriate?  
+
+5. **Predictive validity** – Are the measurements obtained from the measurement model predictive of measurements 
+   of any relevant observable properties or other unobservable theoretical constructs?
+
+6. **Hypothesis validity** – This describes the nature of the hypotheses that could emerge from the measurements 
+   produced by the measurement model, and whether those are “substantively interesting”.
```

Comment:
I have seen the sphinx build struggle with these quotes you're using in this line. Can we replace them with the basic ones `"` that you're using in other parts (like in the citation, for example)?